firefox/data/class_model.js

```
@@ -0,0 +1,278 @@
+            this.event.fire("onTrelloDataReady");
+        }
+        //else log('checkTrelloDataReady: NO');
+    }
+
+    loadTrelloLists(boardId) {
+        log("loadTrelloLists");
+
+        this.trello.lists = null;
+
+        Trello.get(
+            "boards/" + boardId,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)